### DISCUSSION TITLE
Does it change anything to do with firefox?

### DISCUSSION THREAD

**Discussion by mTvare6** [asmvik/yabai#888] [General]
![image](https://user-images.githubusercontent.com/57838468/115735093-d5658c00-a3a7-11eb-9b6d-c46dc47375b6.png)
I finally disabled SIP and injected yabai to dock, And after this I noticed lot of thing changed in firefox, the bars around tabs because of yabai, or something else happened in between?

**Comment by asmvik** [comment]
This could be an issue where tabs report an incorrect accessibility role. Firstly, do you have any rules in your yabairc file?

Otherwise, try to disable window borders: `yabai -m config window_border off` to disable globally, or simply disable just for firefox `yabai -m rule --add app="^Firefox$" border=off`
